src/avram/object_extensions.cr

```diff
@@ -0,0 +1,30 @@
+class Object
+  def blank_for_validates_required? : Bool
```

Comment:
The problem boils down to https://github.com/kostya/blank #blank? returning `true` for arrays and for `Bool` if it is false.

So what I did was add this new method we can use in `validate_required`.

But this makes me wonder if we should fork the `blank` library and just add these in so this behavior is like this for `blank?

The problem is it may be unexpected for users of Rails since Rails `false.blank? # true` and `[].blank? # true`. IMO this behavior is confusing and I'd expect those to return `false` but maybe it's just me!

cc @edwardloveall @jwoertink

Reply:
My thought is that either way I can merge this in for now to fix the linked issue. But would love to figure out if we should just make these changes to `blank?` directly in a later PR